### PR TITLE
Buildwechaty

### PR DESCRIPTION
**Pull request by gangena**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Comment by gangena** [comment]
Corresponding blog here<https://github.com/wechaty/wechaty.js.org/issues/909>

File: jekyll/_posts/2020-04-09-how-to-build-a-chatbot-from-zero-to-one-on-wechaty.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: "如何从0到1搭建微信聊天机器人" 
+author: juzibot
```
Comment (lijiarui) [inline jekyll/_posts/2020-04-09-how-to-build-a-chatbot-from-zero-to-one-on-wechaty.md]: Please change this to `lijiarui`

File: jekyll/_posts/2020-04-09-how-to-build-a-chatbot-from-zero-to-one-on-wechaty.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+categories: announcement
+tags:
+  - summer-of-wechaty
+  - summer-2020
```
Comment (lijiarui) [inline jekyll/_posts/2020-04-09-how-to-build-a-chatbot-from-zero-to-one-on-wechaty.md]: Please remove the useless tag.

Add some useful tag for this, eg: chatbot, global AI.

File: jekyll/_posts/2020-04-09-how-to-build-a-chatbot-from-zero-to-one-on-wechaty.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  - meetup
+  - featured
+image: /assets/2020/04-how-to-build-a-chatbot-from-zero-to-one-on-wechaty/00.webp
+---
+
+> 作者: [juzibot](https://github.com/juzibot/)
+
+## 李佳芮
+
+句子互动的创始人，GitHub上开源项目Wechaty上面的联合作者，Wechaty在github上面有超过1万star，并搭建了一个覆盖全球的微信平台开发者社区，微软的AIWP，出版书：《Chatbot从0到1》，已经在京东上架，欢迎大家阅读，本次分享里面会简单介绍书中的内容，如何从0到1搭建一个聊天机器人，
```
Comment (lijiarui) [inline jekyll/_posts/2020-04-09-how-to-build-a-chatbot-from-zero-to-one-on-wechaty.md]: AIWP--> AI MVP

File: jekyll/_posts/2020-04-09-how-to-build-a-chatbot-from-zero-to-one-on-wechaty.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+- [0:30:31](https://youtu.be/Uqtx6UViEro?list=PL8hd9KDTdarDXf_Rxtr8meKhxtgcXMInh&t=2354) 如何实操搭建一个聊天机器人
+{% include iframe.html src="https://youtu.be/Uqtx6UViEro?list=PL8hd9KDTdarDXf_Rxtr8meKhxtgcXMInh" %}
+
+### 对话系统的趋势
+
+人类一直在寻找更加便捷的人机交互方式，在80年代出现最简单的人机交互方式，如果要完成一个交互需要下载一个应用；后来出现了浏览器、网站等，发展到已经不需要在电脑上下载一个应用了；后来出现了手机，整个交互方式变成了以移动触摸为主的交互，大大提高了效率和便捷性；未来会有更多的交互方式，会变成我说一句话或者一个手势，机器就可以立即把这个任务完成。
+能够发现，我们要的需求一直没变，但是交互方式一直在发生变化，从最早以鼠标键盘为主的电脑，陆续变成移动触摸为主的手机等移动终端，再到未来以自然语言为主的chatbot聊天机器人，交互方式会变得越来越便捷，机器变得像人一样聪明是未来交互的趋势。
+![intro](/assets/2020/04-how-to-build-a-chatbot-from-zero-to-one-on-wechaty/01.webp)
+我们可以看到，原来的机型是超级计算机，体型巨大无比只有专家才能用到，而现在就算是三五岁的小孩子或是年迈老年人，不是很懂机器也可以使用。交互从没有智能到有智能，从无界面到有界面的变化。我们来对对话系统来做个类比，在过去的手机因为有了操作系统，变成了智能手机
+![intro](/assets/2020/04-how-to-build-a-chatbot-from-zero-to-one-on-wechaty/04.webp)
```
Comment (lijiarui) [inline jekyll/_posts/2020-04-09-how-to-build-a-chatbot-from-zero-to-one-on-wechaty.md]: It seems strange the image name you use. Please change to a more meaningful image name.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2020-04-09-how-to-build-a-chatbot-from-zero-to-one-on-wechaty.md (hunk 4 above)
Comment (gangena) [reply jekyll/_posts/2020-04-09-how-to-build-a-chatbot-from-zero-to-one-on-wechaty.md]: new blog at <https://github.com/wechaty/wechaty.js.org/pull/1221>

**Comment by lijiarui** [comment]
duplicate with  #1221

**Comment by huan** [comment]
Close because it's duplicated with an already published blog.

I hope the PR creator can take more responsibility for managing the PRs that they created in the future. @lijiarui

**Pull request closed without merging**